src/GraphDrawer/GraphDrawer.js

```diff
@@ -35,16 +35,6 @@ class GraphDrawer extends Component {
     });
   }
 
-  getPrecipitation() {
```

Comment:
The precip and metrics endpoints are now merged - so this code is removed because precipitation can simply be pulled from one endpoint now